dojo/filters.py

```diff
@@ -1610,6 +1640,7 @@ class FindingFilterHelper(FilterSet):
                 lookup_expr="isnull",
                 exclude=True,
                 label="Has Group JIRA")
+            has_any_jira = FindingHasJIRAFilter(label="Has Any JIRA")
```

Comment:
```suggestion
        has_any_jira = FindingHasJIRAFilter(label="Has Any JIRA")
```
This should be present if groups are enabled or not, since regular jira links are a part of this